packages/storybook/examples/core/Tooltip.stories.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useRef } from 'react';
+
+import Tooltip, { PureTooltip } from '@ichef/gypcrete/src/Tooltip';
+
+export default {
+    title: '@ichef/gypcrete|Tooltip',
+    component: PureTooltip,
+    subcomponents: {
+        'renderToLayer(anchored({})': Tooltip,
+    },
+};
+
+export function BasicUsage() {
+    return (
+        <div>
+            <PureTooltip>tooltip</PureTooltip>
+
+            <div style={{ height: 30 }} />
+
+            <PureTooltip placement="bottom">
+                placed at bottom of target
+            </PureTooltip>
+
+            <div style={{ height: 30 }} />
+
+            <PureTooltip arrowStyle={{ left: '12px' }}>
+                custom arrow style
+            </PureTooltip>
+        </div>
+    );
+}
+
+export function AnchoredTooltip() {
```

Comment:
Rewrite old anchored Tooltip class component as function component to read example implementation code directly.